.github/workflows/main.yml

```
@@ -93,21 +93,24 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
-
+      - name: Get dbt-core version
+        # if this is a pull request uses ref of base branch otherwise uses ref of current commit
+        id: dbt-core-version
+        run: echo "::set-output name=dbt-version::${{ github.event_name == 'pull_request' && github.base_ref || github.ref }}"
       - name: Install python dependencies
         run: |
           pip install  --user --upgrade pip
           pip install tox
           pip --version
           tox --version
 
-      - name: Install dbt-core latest
+      - name: Install dbt-core from branch ${{ steps.dbt-core-version.outputs.dbt-version.dbt-core-ref }}
```

```suggestion
      - name: Install dbt-core from branch ${{ steps.dbt-core-version.outputs.dbt-version }}
```
I don't think you need the `dbt-core-ref` anymore do you?

---

name change was based on suggestion by @kwigley in earlier comments https://github.com/dbt-labs/dbt-redshift/pull/80#discussion_r830368205

---

Name change modified in steps description, postgres section and steps to all be assigned same.